src/app/login/login.component.ts

```diff
@@ -55,8 +62,9 @@ export class LoginComponent {
     if (this.validateFields()) {
       try {
         const result: SLLoginResult = await this.authService.login(this.email, this.password);
+        const captchaResult: SLCaptchaResult = await this.authService.captcha(this.captcha);
 
-        if (result.success) {
+        if (result.success && captchaResult.success) {
```

Comment:
I put this here, but we can take it out. The only reason this is here right now is because I haven't really made the recaptcha pop up on failed attempts yet